packages/commonwealth/client/scripts/views/components/Knock/Knock.css

```diff
@@ -0,0 +1,4 @@
+.knock-notifications-button svg {
+  color: rgb(117, 117, 117);
+  fill: rgb(255, 255, 255) !important;
+}
```

Comment:
we should remove blue border from focused elements

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/d82631e2-4523-4f56-adf7-e44b4c0383b4)

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/a872c585-03ab-4083-a12c-07cbf6c40f35)